### PR TITLE
Added interaction snippets #7

### PR DESCRIPTION
**Pull request by bucurdavid**

### CODE REVIEW

**Comment by bucurdavid** [comment]
Below is the contract will all interactions:
https://devnet-explorer.elrond.com/accounts/erd1qqqqqqqqqqqqqpgqcujn96cu3a0ryq6tg66sgxat4hszpl006h7scu4j3r

File: interaction/devnet.snippets.sh [hunk 1]
```diff
@@ -1,233 +1,266 @@
...
+    --pem=${WALLET} \
+    --gas-limit=6000000 \
+    --function "setAdministrator" \
+    --arguments $address \
+    --proxy ${PROXY} \
+    --chain ${CHAIN_ID} \
+    --send || return
+}
+
+mintTokenUsingEsdt(){
```
Comment (damienen) [inline interaction/devnet.snippets.sh:236]: Everything seems good. The only thing missing is a mintTokenUsingEgld method

**Review by damienen — CHANGES_REQUESTED** [review]
Add mintTokenUsingEgld

**Comment by bucurdavid** [comment]
Same link you can see the new **mintTokenUsingEgld** method

**Pull request merged**